docs/src/content/docs/reference/window/window.mdx

```diff
@@ -48,6 +48,11 @@ The window icon shown in the title bar or the task bar on window managers suppor
 Whether the window should be borderless/frameless or not.
 </SlintProperty>
 
+### skip-taskbar
+<SlintProperty propName="skip-taskbar" typeName="bool" defaultValue="false">
+Whether to show or hide the window icon in the taskbar. (only supported on `Windows`)
```

Comment:
I can't find any other use of Windows in backticks in the docs, so proposing an alternative:

```suggestion
Whether to show or hide the window icon in the taskbar. This is feature only supported on Windows.
```